internal/pkg/service/buffer/worker/service/file_import.go

```diff
@@ -55,19 +52,22 @@ func (s *Service) importFiles(ctx context.Context, wg *sync.WaitGroup, d depende
 					}
 				}()
 
+				// Load token
+				token, err := s.store.GetToken(ctx, fileRes.ExportKey)
+				if err != nil {
+					return "", errors.Errorf(`cannot load token for export "%s": %w`, fileRes.ExportKey, err)
+				}
+
+				api, err := keboola.NewAPI(ctx, s.storageAPIHost, keboola.WithClient(&s.httpClient), keboola.WithToken(token.Token))
+				if err != nil {
+					return "", err
+				}
+
+				defer s.events.SendFileImportEvent(ctx, api, time.Now(), &err, fileRes)
```

Comment:
Super, bolo by dobre to nejak pokryt testom.

Napriklad pridat `assert.Length(t, debugLogger.Warnings(), 0)` + komentar co to testuje.

Slice upload:
- https://github.com/keboola/keboola-as-code/blob/4442e17a2bdb274ed6564d86ecb1d5525c6b7c6e/internal/pkg/service/buffer/worker/service/slice_upload_test.go#L215

File import:
- https://github.com/keboola/keboola-as-code/blob/4442e17a2bdb274ed6564d86ecb1d5525c6b7c6e/internal/pkg/service/buffer/worker/service/file_test.go#L109